app/views/directives/deploy-image.html

```diff
@@ -73,7 +73,8 @@
         administrator.
       </div>
       <h2>
-        {{app.name}}<span ng-if="import.tag">:{{import.tag}}</span>
+        <span ng-if="mode === 'dockerImage'">{{app.name}}<span ng-if="import.tag">:{{import.tag}}</span></span>
```

Comment:
hmm does it make sense to be changing that header at all? isn't that header the name and metadata of the image?